client/app/routes/landuse-form.js

```diff
@@ -27,6 +28,7 @@ export default class LanduseFormRoute extends Route.extend(AuthenticatedRouteMix
     return RSVP.hash({
       package: landuseFormPackage,
       accounts: await this.store.findAll('account'),
+      zoningResolutions: await this.store.findAll('zoning-resolution'),
```

Comment:
hmm.. it looks like you are both side loading zoning-resolutions along with landuse-form.landuse-actions (line 21) and also querying for them directly here?

Reply:
I know I explained this in the chat but just reposting here for documentation purposes :) 

We need to load zoning resolutions in two different ways. One is for the dropdown that's displayed on the Proposed Actions section. This dropdown needs to list EVERY possible zoning resolution (around 600) that a user could choose from. The second way -- we need to know which zoning resolutions are tied to which landuse-actions so that when the form is reloaded, we can show which zoning resolution was selected and saved previously for that specific landuse-action. If we only got zoning resolutions through the `expand`, we'd have only zoning resolutions associated with the landuse-actions on the page, and therefore wouldn't be able to populate the dropdown.